packages/ag-charts-enterprise/src/features/error-bar/errorBar.test.ts

```diff
@@ -602,4 +602,29 @@ describe('ErrorBars', () => {
         await clickAction(x + width - 1, y)(chart);
         await compare();
     });
+
+    it('should use correct cursor', async () => {
+        const getCursor = () => {
+            return chart['cursorManager']['element'].style.cursor;
+        };
```

Comment:
Seems a little fragile, but I'm ok with this as long as we don't repeat it elsewhere (or have one place we define this) - could you move this to a test utility function, perhaps as a sibling to where `hoverAction()` is declared?

I think it might be better to also use a cast as early as possible so that trivial refactors highlight a break here?

e.g. `(chart as Chart).getModuleContext().cursorManager.element?.style.cursor` might be a better way to go (make `element` a `public readonly` field on `CursorManager` for test purposes perhaps?

Reply:
I've replaced this with `chart.getModuleContext().cursorManager.getCursor()`